tests/conformance/scripts/run-tests/runners/stringifyRecords.ts

```diff
@@ -29,19 +29,8 @@ function normalizeRecord(record: McapTypes.TypedMcapRecord) {
     .value();
 }
 
-export function stringifyRecords(
-  records: McapTypes.TypedMcapRecord[],
-  variant: TestVariant,
-): string {
-  const normalizedRecords = records.map(normalizeRecord);
-  const features = Array.from(variant.features.values());
-  return (
-    stringify(
-      {
-        records: normalizedRecords,
-        meta: { variant: { features } },
-      },
-      { replacer },
-    ) + "\n"
-  );
+export function toSerializableMcapRecord(
+  record: McapTypes.TypedMcapRecord,
+): SerializableMcapRecord {
+  return JSON.parse(JSON.stringify(normalizeRecord(record), replacer)) as SerializableMcapRecord;
```

Comment:
Maybe don't need to roundtrip stringify/parse here?

Reply:
the other alternative is to do the recursive replace myself, which i'm not perfectly confident in how to do in JS without hitting a corner case